launcher/src/main.rs

```diff
@@ -1528,37 +1621,101 @@ fn spawn_shards(
     Ok(())
 }
 
+#[derive(Debug)]
+enum Gpu {
+    RTX4090,
+    T4,
+    L4,
```

Comment:
We probably want L40 as well, since we are using them now?

Reply:
Done.